main/src/org/destinationsol/game/planet/PlanetManager.java

```diff
@@ -60,10 +60,10 @@ public PlanetManager(TextureManager textureManager, HullConfigManager hullConfig
         mySysConfigs = new SysConfigs(textureManager, hullConfigs, itemManager);
         myMazeConfigs = new MazeConfigs(textureManager, hullConfigs, itemManager);
 
-        mySystems = new ArrayList<SolSystem>();
-        myMazes = new ArrayList<Maze>();
-        myPlanets = new ArrayList<Planet>();
-        myBelts = new ArrayList<SystemBelt>();
+        mySystems = new ArrayList<>();
```

Comment:
No, it won't.

The reason I explicitly make it a point to remove the type detail is, CheckStyle complains about it (Java 7.0 and above don't need explicit types, since it can be deduced from the type of myPlanets). Both are identical, the reduced down version is just cleaner. Code simplification, less warnings, I'm happy with that :D
Although if you want, I can revert back - not that I recommend it.

And no, it won't complain about generics. I've removed the type info in almost every class I've touched AND have converted all for loops to foreach loops ( `Planet planet : myPlanets` ) and yet it works perfectly :)